src/fides/api/schemas/connection_configuration/connection_secrets_postgres.py

```diff
@@ -17,6 +17,7 @@ class PostgreSQLSchema(ConnectionConfigSecretsSchema):
         str
     ] = None  # Either the entire "url" *OR* the "host" should be supplied.
     port: Optional[int] = None
+    ssh_required: bool = False
```

Comment:
Is there a better name or description to be applied here?

Reply:
This is great